apps/subscription/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from apps.user.models import User
+
+# Create your models here.
+
+
+# Subject Data Model
+class Subject(models.Model):
+    name = models.CharField(max_length=100)
```

Comment:
I like the idea of a normalized subject model. Do you think this should be unique and indexed?